ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -284,23 +284,34 @@ open class BottomCommandingController: UIViewController {
     public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
 
-        if previousTraitCollection?.horizontalSizeClass != traitCollection.horizontalSizeClass {
+        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+            updateSheetPreferredExpandedContentHeight()
+        }
+    }
+
+    public override func willTransition(to newCollection: UITraitCollection, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.willTransition(to: newCollection, with: coordinator)
+
+        if newCollection.horizontalSizeClass != traitCollection.horizontalSizeClass && newCollection.userInterfaceIdiom == .pad {
```

Comment:
For context in the future, could you add a comment on why this this is only needed on the iPad? (like you did in the description of the PR)

`Make sure we only reset our commanding style on iPads. On iPhone we only use the sheet style, so there's no need for unnecessary work.`